x11-misc/autocutsel/autocutsel-0.10.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+DESCRIPTION="Synchronise the two copy/paste buffers mainly used by X applications"
+HOMEPAGE="http://www.nongnu.org/autocutsel/"
+SRC_URI="https://github.com/sigmike/${PN}/releases/download/${PV}/${P}.tar.gz"
+IUSE=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc sparc x86"
+
+RDEPEND="x11-libs/libX11
+	x11-libs/libXmu
+	x11-libs/libXt
+	x11-libs/libXaw
+	x11-libs/libXext"
+DEPEND="${RDEPEND}
+	x11-proto/xproto"
+
+src_install() {
```

Comment:
in case EAPI=5 is added above, I think this whole phase definition is obsolete and already implicit